cpu/cortexm_common/thread_arch.c

```diff
@@ -234,6 +238,15 @@ char *thread_arch_stack_init(thread_task_func_t task_func,
     return (char*) stk;
 }
 
+/* This function returns the number of bytes used on the ISR stack */
+int thread_arch_isr_stack_usage(void)
+{
+    uint32_t  *ptr = &_sstack;
+    while (*(ptr++) == STACK_CANARY_WORD) {
+    }
+    return ISR_STACKSIZE - (ptr - &_sstack);
```

Comment:
just a quick question, the AVR and the Atmega CPUs don't have an ``` ISR_STACKSIZE ``` defined in either of them, i thought of adding a dummy define of value zero, would it be ok for now?
I would need the ISR_STACKSIZE defined in every CPU since i use its value in the  ``` ps.c ``` .